projects/packages/jitm/src/class-jitm.php

```diff
@@ -36,7 +48,7 @@ public static function configure() {
 	 * @return Post_Connection_JITM|Pre_Connection_JITM JITM instance.
 	 */
 	public static function get_instance() {
-		if ( ( new Connection_Manager() )->is_connected() ) {
+		if ( ( new Connection_Manager() )->is_connected() || defined( 'IS_WPCOM' ) && IS_WPCOM ) {
```

Comment:
I don't like this pattern much. Should Connection_Mananger->is_connected() always return true on WPCOM?

Reply:
Also is this change required as part of this PR?

Reply:
I don't think it's required yet no. I'll remove it. Good spot 👍